codalab/lib/bundle_cli.py

```diff
@@ -1192,6 +1208,119 @@ def do_run_command(self, args):
         self.wait(client, args, new_bundle['uuid'])
 
     @Commands.command(
+        'run-local',
+        help='Run a run bundle locally.',
+        arguments=(
+            Commands.Argument('target_spec', help=ALIASED_TARGET_SPEC_FORMAT, nargs='*', completer=TargetsCompleter),
+            Commands.Argument('command', metavar='[---] command', help='Arbitrary Linux command to execute.', completer=NullCompleter),
+            Commands.Argument('-w', '--worksheet-spec', help='Operate on this worksheet (%s).' % WORKSHEET_SPEC_FORMAT, completer=WorksheetsCompleter),
+        ) + Commands.metadata_arguments([RunBundle]) + EDIT_ARGUMENTS + WAIT_ARGUMENTS,
+    )
+    def do_run_local_command(self, args):
+        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec)
+        args.target_spec, args.command = cli_util.desugar_command(args.target_spec, args.command)
+        targets = self.parse_key_targets(client, worksheet_uuid, args.target_spec)
+        metadata = self.get_missing_metadata(RunBundle, args)
+
+        docker_image = metadata.get('request_docker_image', 'ubuntu:14.04')
```

Comment:
I think If people are doing run-local, the `--request-docker-image` option should be required (they should be explicit about what they're doing.)  99% of the invocations will not be with `ubuntu:14.04`.